### DISCUSSION TITLE
Create Assignement not working

### DISCUSSION THREAD

**Discussion by soumayasassi** [community/community#47241] [GitHub Education]
Hello, 
It has been two weeks that i can not create an assignement on my githubClassroom.
I have already created one assignement previously. 
I obtain this error , every time i try to create an assignement on all my githubclassroom.
Can you please help me? 
![error](https://user-images.githubusercontent.com/45098633/218468184-12d7807e-9374-4ddc-91e6-c9ad7216e740.png)

**Comment by samus-aran** [comment]
Hi @soumayasassi 👋 

Thank you so much for bringing this to our attention! We'd passed this onto the internal team. If we could ask you to retry creating a new assignment the issue should be resolved. If you are still experiencing errors please let us know 👍
